content/300-accelerate/400-api-reference.mdx

```diff
@@ -113,3 +113,9 @@ interface AccelerateInfo {
 | `region`       | `String`                                                     | The data center region that received the request.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | `requestId`    | `String`                                                     | Unique identifier of the request. Useful for troubleshooting.                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | `signature`    | `String`                                                     | The unique signature of the Prisma operation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+
+## Errors
+
+Accelerate-related errors start with `P6xxx`.
```

Comment:
this line is quoted from the [actual error reference](https://www.prisma.io/docs/orm/reference/error-reference#accelerate):

<img width="515" alt="image" src="https://github.com/prisma/docs/assets/4058327/0056fa78-f1d8-44da-8e70-5fa8a90d5ddc">

Reply:
LGTM 👍 , maybe both places should read `Prisma Accelerate` rather than Accelerate.

Reply:
changed "Accelerate" to "Prisma Accelerate" on the other page as well: https://github.com/prisma/docs/pull/5577